### DISCUSSION TITLE
SDXL TextEncoder 1 & 2

### DISCUSSION THREAD

**Discussion by caniyabanci76** [kohya-ss/sd-scripts#918] [Q&A] [answered]
Does TE1 = CLIP G and TE2 = CLIP L ?

**Comment by kohya-ss** [comment, marked as answer]
Sorry for lacking of the documentation. Text Encoder 1 = ViT-L (768 dims) and Text Encoder 2 = BiG-G (1280 dims).

This is because SDXL state dict has `conditioner.embedders.0` keys for ViT-L and `conditioner.embedders.1` keys for BiG-G.
